Build/setup_glass.py

```diff
@@ -306,9 +309,9 @@ def generate_python_version_props(build_output_dir: str | None):
 def main(build_output_dir: str | None = None):
     get_drop_exe()
     get_glass()
+    copy_glass()
     get_test_console_app()
     copy_ptvs_output(build_output_dir)
-    generate_python_version_props(build_output_dir)
```

Comment:
deleted 'generate_python_version_props' here because 'copy_ptvs_output' step is also calling it